packages/react-scripts/template/src/serviceWorker.js

```diff
@@ -89,6 +82,18 @@ function registerValidSW(swUrl, config) {
           }
         };
       };
+      function newerSwAvailable(){
+        // At this point, the updated precached content has been fetched,
+        // but the previous service worker will still serve the older
+        // content until all client tabs are closed.
+        console.log(
+          'New content is available and will be used when all ' +
+            'tabs for this page are closed. See http://bit.ly/CRA-PWA.'
```

Comment:
Neat PR!

This `bit.ly` link needs to be pointed to `https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app` now. I created `http://bit.ly/CRA2-PWA`  but not sure if @gaearon wants to do it from his own account or something similar :)

Reply:
There are some issues with updating these links as these files (with the original links) are present in existing projects. More details here: https://github.com/facebook/create-react-app/issues/5536

Reply:
ah, I see. Thanks.

Reply:
done!